dashboard/src/shared/PackageRepositoriesService.ts

```diff
@@ -223,14 +223,30 @@ export class PackageRepositoriesService {
     // An "Any" object has "typeUrl" with the FQN of the type and a "value",
     // which is the result of the encoding (+finish(), to get the Uint8Array)
     // of the actual custom object
+    let detail, helmCustomDetail: HelmPackageRepositoryCustomDetail;
```

Comment:
Both `detail` and `helmCustomDetail` seem to be scoped to the `PluginNames.PACKAGES_HELM` case block. Why not declaring it inside, instead of outside the `switch` statement?

Reply:
I was simply getting a linter error (https://eslint.org/docs/latest/rules/no-case-declarations), but after a more thorough reading, I can just add brackets around the `case:  { ........ }` to avoid the linter issue. Will do it (and check similar issues in the code) straight away! Thanks